static/js/settings_users.js

```diff
@@ -428,19 +428,60 @@ function get_human_profile_data(fields_user_pills) {
 }
 
 export function confirm_deactivation(user_id, handle_confirm, loading_spinner) {
+    const bots_owned_by_user = bot_data.get_all_bots_owned_by_user(user_id);
+    let bot_text;
+    if (bots_owned_by_user.length === 0) {
+        bot_text = $t_html({defaultMessage: "does not administer any active bots."});
+    } else {
+        bot_text = $t_html({defaultMessage: "administers the following active bots:"});
+    }
+
     const user = people.get_by_user_id(user_id);
     const opts = {
         username: user.full_name,
         email: settings_data.email_for_user_settings(user),
+        bot_text,
+        bots_owned_by_user,
     };
     const html_body = render_settings_deactivation_user_modal(opts);
 
+    function fill_number_of_invites_by_user() {
+        // We display 0 number of invites as the default. Knowing the number of invites
+        // requires making this request. If it fails, we do nothing.
```

Comment:
Let's move this comment after the `success` handler inside the `channel.get` argument block -- that's naturally where one would look for an error handler.